web/packages/teleport/src/Discover/Database/CreateDatabase/CreateDatabase.tsx

```diff
@@ -337,14 +337,15 @@ const CreateDatabaseDialog = ({
   );
 };
 
-// PORT_REGEXP only allows digits with length 4.
-export const PORT_REGEX = /^\d{4}$/;
+// only allows digits with valid port range 1-65535. Not using RegEx as more effienct to use logic comparsions.
```

Comment:
```suggestion
// only allows digits with valid port range 1-65535
```

This gets rid of a typo in effienct, and removes unnecessary info.